yente/search/indexer.py

```diff
@@ -202,3 +213,99 @@ async def update_in_thread() -> None:
     )
     thread.start()
     # asyncio.to_thread(update_index, force=force)
+
+
+class DeltasNotAvailable(Exception):
+    pass
+
+
+async def get_deltas_from_version(
+    version: str, dataset: Dataset
+) -> AsyncGenerator[str, None]:
+    """
+    Get deltas from a specific version of a dataset.
+    """
+    version = version.replace(settings.INDEX_VERSION, "")
```

Comment:
Should be factored out to avoid falling apart once we make a change to versions in some other place.

Reply:
+1